src/main/java/Information.java

```diff
@@ -0,0 +1,4 @@
+public class Information {
+    Task List[] = new Task[100];
+    int listLength = 0;
```

Comment:
I like the use of Task as a list in its own class instead of in Duke! Makes the code much easier to read!